packages/toolkit/src/utils.ts

```diff
@@ -23,6 +23,10 @@ It is disabled in production builds, so you don't need to worry about that.`)
   }
 }
 
+export function delay(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms))
+}
+
```

Comment:
*cough*

https://github.com/reduxjs/redux-toolkit/blob/988c0499125c07e203794e6978e4796447c56579/packages/toolkit/src/query/tests/helpers.tsx#L39-L41

Maybe we should just rename that one? 😅